test/logger-usage/build.gradle

```diff
@@ -27,8 +27,7 @@ dependencies {
 
 loggerUsageCheck.enabled = false
 
-forbiddenApisMain.enabled = true // disabled by parent project
```

Comment:
Do we need to keep this?

Reply:
No, I looked at the parent but did not see it being disabled there. Also, using onlyIf now, there would be no way to disable (using onlyIf) in the parent but then re-enable in the child project.